Upgrade-Guide/op_version.md

```diff
@@ -10,7 +10,11 @@ After Gluster upgrade, it is advisable to have op-version updated.
 
 Current op-version can be queried as below:
 
-    [root@~]#gluster volume get  <volume name> cluster.op-version
+    [root@~]#gluster volume get all cluster.op-version
+
+To check maximum op-version supported by the cluster, the following query can be used:
```

Comment:
This should be, "to get maximum op-version that the cluster can support" right? I mean is one node can support X+ and another X then this returns X.